purchase_requisition_analytic_rfq/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Moduon - Andrea Cattalani
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl
+{
+    "name": "Purchase requisition analytic rfq",
+    "summary": "Add analytic account in RFQ lines",
+    "version": "14.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["shide", "anddago78"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "analytic",
+        "sale_purchase_requisition_analytic",
```

Comment:
This dependency has to change:

```suggestion
        "purchase_requisition",
```